test-loop-tests/src/utils/account.rs

```diff
@@ -0,0 +1,55 @@
+use near_chain_configs::test_genesis::ValidatorsSpec;
+use near_primitives::types::AccountId;
+
+pub fn create_account_id(s: &str) -> AccountId {
+    s.parse().unwrap()
+}
+
+pub fn create_account_ids<const N: usize>(s: [&str; N]) -> [AccountId; N] {
+    s.map(|s| create_account_id(s))
+}
```

Comment:
is it worth to make this one generic? I see that is some places it is used as `Vec` and some have `static[]`
```suggestion
pub fn create_account_ids<I, T>(s: I) -> T 
where 
    I: IntoIterator<Item = &str>,
    T: FromIterator<AccountId>
{
    s.into_iter().map(|s| create_account_id(s)).collect()
}
```